src/styles/claudy.css

```diff
@@ -0,0 +1,34 @@
+[role=banner] .coz-claudy {
+  position: fixed;
+  bottom: 5em;
+  right: 2em;
+}
+
+@media (min-width: 64em) {
+  [role=banner] .coz-claudy {
+    bottom: 2em;
+  }
+}
+
+[role=banner] .coz-claudy .coz-claudy-icon {
```

Comment:
Why not just `[role=banner] .coz-claudy-icon` as the selector? 
Unless you really need extra specificity weight, which I doudt, the naming convention should be enough to understand that `coz-claudy-icon` is a child of `coz-claudy` I think.